src/passes/Outlining.cpp

```diff
@@ -38,16 +240,16 @@ struct Outlining : public Pass {
                            const SuffixTree::RepeatedSubstring& substring,
                            const std::vector<Expression*>& exprs) {
     auto startIdx = substring.StartIndices[0];
-    // The outlined functions can be named anything. Using the start index of
-    // the first time the outlined sequence was seen in the module may help
-    // with debugging later
+    // The outlined functions can be named anything. Use the start index of
+    // the first time the outlined sequence was seen in the module to help
+    // with debugging later.
```

Comment:
nittiest of nits: It would be good to add periods to the other comments as well ;)

Reply:
done